### PR TITLE
Setup database schema and corresponding models.

### LINKED ISSUE #2: Setup Travis CI

### PR DESCRIPTION
**Pull request by begor**
Also:
- Add basic test suites for models (by and large for Travis and self contentment)
- Setup Travis CI builds.

Resolves #2 and #3.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - postgresql
+
+elixir:
+  - 1.3.2
+otp_release:
+  - 19.0
+
+notifications:
+  recipients:
+    - birykovegor@gmail.com
```
Comment (jomei) [inline .travis.yml]: Only you? What's bestpractice here? May be we should send email on `lanceorg@gmail.com`  or something like that?

File: config/dev.exs [hunk 2]
```diff
@@ -36,8 +36,8 @@ config :phoenix, :stacktrace_depth, 20
 # Configure your database
 config :cb, CB.Repo,
   adapter: Ecto.Adapters.Postgres,
-  username: "postgres",
-  password: "postgres",
-  database: "cb_dev",
+  username: "cb",
+  password: "qwerty",
+  database: "cb",
```
Comment (jomei) [inline config/dev.exs]: Our future test will need clear test db for each test run, so lets use `cb_dev` for development environment and `cb_test` for test one

File: priv/repo/migrations/20161001123104_create_task.exs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  use Ecto.Migration
+
+  def change do
+    create table(:tasks) do
+      add :title, :string
+      add :description, :string
+      add :photos, {:array, :string}
+      add :tags, {:array, :string}
+      add :location, :string
+      add :executor_id, references(:users, on_delete: :delete_all)
```
Comment (jomei) [inline priv/repo/migrations/20161001123104_create_task.exs]: `on_delete: :delete_all` - that means "delete all users when deleting task" or "delete all tasks when deleting user"?

dont read documentation @ ask Egor about every single line

File: .travis.yml (hunk 1 above)
Comment (begor) [inline .travis.yml]: Yes, I think something generalized, like `lanceorg@gmail.com` will work better. 
If you are sure about `lanceorg` (btw, what it means?), I can change it.

File: config/dev.exs (hunk 2 above)
Comment (begor) [inline config/dev.exs]: It's already kinda like this.
I will change `cb` to `cb_dev`, ok.

File: priv/repo/migrations/20161001123104_create_task.exs (hunk 3 above)
Comment (begor) [inline priv/repo/migrations/20161001123104_create_task.exs]: From Ecto docs:
> :on_delete - The action taken on associations when parent record is deleted.

If a user is deleted, so the job is.

All this `on_delete` strategies picked in a somewhat random fashion, I guess it's better be decided more carefully (maybe later, when we'll gain better understanding of relations).

**Pull request merged**